datalake/constructs/batch_job_queue.py

```diff
@@ -67,15 +66,27 @@ def __init__(
         cloudformation_launch_template = aws_ec2.CfnLaunchTemplate(
             self,
             "batch-launch-template",
-            launch_template_name="datalake-batch-launch-template",
+            launch_template_name=f"{deploy_env}-datalake-batch-launch-template",
             launch_template_data=launch_template_data,
         )
         launch_template = aws_batch.LaunchTemplateSpecification(
             launch_template_name=cloudformation_launch_template.launch_template_name
         )
 
+        # use existing VPC in LINZ AWS account.
+        # VPC with these tags is required to exist in AWS account before being deployed.
+        # A VPC will not be deployed by this project.
+        datalake_vpc = aws_ec2.Vpc.from_lookup(
```

Comment:
The "datalake" prefix is probably redundant, since the code is inside the context of the datalake.